corehq/apps/accounting/models.py

```diff
@@ -3736,7 +3736,8 @@ def _get_stripe_customer(self):
     @property
     def all_cards(self):
         try:
-            return [card for card in self.customer.cards.data if card is not None]
+            cards = stripe.Customer.list_sources(customer=self.customer.id, object="card")
+            return [card for card in cards.data if cards is not None]
```

Comment:
This will fail if `cards` is `None` when it will try to access `cards.data`.

Reply:
I think @AmitPhulera was pointing out a typo here.
```suggestion
            return [card for card in cards.data if card is not None]
```